src/data.js

```diff
@@ -15,46 +15,124 @@ export const quizData = {
 
   questions: [
     {
-      text: 'What are the different ways to declare a JS variable?',
+      text: 'Why orange is the national colour of the Netherlands?',
       answers: {
-        a: 'constant, let, variable',
-        b: 'var, const, let, function',
-        c: 'var, let, const',
+        a: 'it is the color of the sunset over the North sea',
+        b: 'it is the color of Royal Dutch line, the house of Orange',
+        c: 'it is the favorite color of the Queen',
+      },
+      correct: 'b',
+      selected: null,
+      isAnswerCorrect: true,
```

Comment:
I think it wouldn't be wise to use `true` as the default value. It might be better to store what answer the user chose. Like `userAnswer: 'a'`, and then when you refresh the page, you know which answer user chose. Because otherwise, how does the application know to add `correct` or `wrong` class names to relevant options when the user refreshes the page?